api/v1alpha2/bundledeployment_types.go

```diff
@@ -45,6 +45,7 @@ const (
 	ReasonReconcileFailed           = "ReconcileFailed"
 	ReasonUnhealthy                 = "Unhealthy"
 	ReasonUpgradeFailed             = "UpgradeFailed"
+	ReasonPreflightFailed           = "PreflightFailed"
```

Comment:
Do we need another reason, or can we just use `InstallFailed` and `UpgradeFailed`, etc?

Reply:
Good question! I think we could get away with just using `InstallFailed` and `UpgradeFailed`.

Reply:
Updated in the latest push